### ISSUE TITLE
UnhandledPromiseRejectionWarning: CssSyntaxError: <css input>:1:17: Unknown word

### ISSUE THREAD

**Issue by benoliver999** [mhanberg/jekyll-postcss#11]
Getting the following on version `0.3.1` (and no CSS is generated as a result). This is in a production ENV.

Anyone else seeing this? I am using it alongside tailwind and jekyll-purgecss.

```
(node:5614) UnhandledPromiseRejectionWarning: CssSyntaxError: <css input>:1:17: Unknown word
    at Input.error (/home/ben/bfoliver.com/node_modules/purgecss/node_modules/postcss/lib/input.js:130:16)
    at Parser.unknownWord (/home/ben/bfoliver.com/node_modules/purgecss/node_modules/postcss/lib/parser.js:563:22)
    at Parser.decl (/home/ben/bfoliver.com/node_modules/purgecss/node_modules/postcss/lib/parser.js:204:37)
    at Parser.other (/home/ben/bfoliver.com/node_modules/purgecss/node_modules/postcss/lib/parser.js:166:12)
    at Parser.parse (/home/ben/bfoliver.com/node_modules/purgecss/node_modules/postcss/lib/parser.js:77:16)
    at Function.parse (/home/ben/bfoliver.com/node_modules/purgecss/node_modules/postcss/lib/parse.js:17:12)
    at x.getPurgedCSS (/home/ben/bfoliver.com/node_modules/purgecss/lib/purgecss.js:1:6739)
    at async /home/ben/bfoliver.com/node_modules/purgecss/bin/purgecss:58:18
```

**Comment by mhanberg** [comment]
hi, can you try updating to 0.3.2 and seeing if this still happens? Thanks!

**Comment by benoliver999** [comment]
Fixed! Apologies, `0.3.2` came out while I was testing this and compiling this issue - you beat me to it!

**Comment by mhanberg** [comment]
No worries, thanks for taking the time to submit a bug report!
